pegasus/routes/v2_section_routes.rb

```diff
@@ -1,4 +1,5 @@
 # Get the set of sections owned by the current user
+# DEPRECATED: Use GET /dashboardapi/sections instead
```

Comment:
What does it mean for this to be deprecated? Are we planning to remove the route at some point?